salesagility/SuiteCRM
# Update tab_panel_content.tpl

Fixed #3763.
Set tabindex in EditView fields as specified by users in Studio

## Description
I have replaced the $tabindex var by the $var.tabindex.
$var is $subfields the first time and $colData.field next three times

## Motivation and Context
EditViews where not usable if you couldn't move with your tab key, point every field with your mouse before start typing all the time is a real pain.

## How To Test This
You can set tabindex for several fields for an EditView in any module in Layouts studio and then navigate to that form. And move with your tab key to see each field selected.

## Types of changes

### Final checklist
